code/modules/mining/mine_items.dm

```diff
@@ -1286,7 +1286,7 @@ var/list/total_extraction_beacons = list()
 		target = get_atom_on_turf(src)
 	if(!target)
 		target = src
-	target.cut_overlay(image_overlay, TRUE)
+	QDEL_NULL(effect_overlay)
```

Comment:
Why not
```suggestion
	cut_overlay(effect_overlay)
```

Reply:
the overlay is in the form of an object, now